records/config.go

```diff
@@ -174,8 +175,8 @@ func SetConfig(cjson string) (c Config) {
 	logging.Verbose.Println("   - SOAExpire: ", c.SOAExpire)
 	logging.Verbose.Println("   - SOAExpire: ", c.SOAMinttl)
 	logging.Verbose.Println("   - RecurseOn: ", c.RecurseOn)
-	logging.Verbose.Println("   - HttpPort: ", c.HttpPort)
```

Comment:
Maybe. But I didn't think it mattered in the case of the logs. But perhaps people are integrating with them. So, I'll change this.